src/utils_project.js

```diff
@@ -284,8 +291,9 @@ go.utils_project = {
                         'Connection': ['close']
                     }
                 });
+
                 return http
-                    .head(voice_url)
+                    .head(voice_url[0])
```

Comment:
@imsickofmaps for clarification, do you mean "if length == 1" (no error) check voice_url[0], and "if length > 1" (error) check voice_url[1]...?